rhscl-postgresql-apb/README.md

```diff
@@ -0,0 +1,25 @@
+rhscl-postgresql-apb
+======================
+
+An apb for deploying [PostgreSQL](https://www.postgresql.org).
+
+## What it does
+* Deploys 1 postgresql container.
+
+## Requirements
+
+## Parameters
+* NAMESPACE: Optional, default 'postgresql-demo-apb', Namespace to deploy the cluster in.
+* POSTGRESQL_DATABASE, default 'admin', Postgresql database name.
+* POSTGRESQL_PASSWORD, default is a randomly generated string, Postgresql database password.
```

Comment:
@rthallisey I am clear with what you are saying, but I think @fabianvf was suggesting passing it in as --extra-vars rather and environment variables whether the command line or from a file.

It has a nice added effect that if it works this way it should work via the broker. We've had occasions where an apb worked when passing in env vars but not extra-vars because it was not properly accounted for. I'd be fine with doing everything in our power to discourage env vars and go with extra-vars.